.github/workflows/sonar.yml

```
@@ -0,0 +1,48 @@
+name: Flutter SonarCloud Analysis
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)